queue-manager/core/src/persistor.ts

```diff
@@ -22,7 +21,7 @@ class Persistor {
   db: Promise<IDBPDatabase<Database>>;
   constructor() {
     this.db = import('idb')
-      .then((idb) => {
+      .then(async (idb) => {
```

Comment:
Why `async`?

Reply:
This is one of the items that are automatically placed after saving the file